cosmo_tester/resources/blueprints/snapshots-migration-321-to-33/openstack-manager-blueprint.yaml

```diff
@@ -0,0 +1,493 @@
+tosca_definitions_version: cloudify_dsl_1_0
```

Comment:
What would be better is to wget this file and patch it like Pawel has done in his test:

https://github.com/codilime/cloudify-system-tests/blob/efa0f9eb886679f2c095eaa4b59dee2dc070a40b/cosmo_tester/test_suites/test_snapshots/snapshots_multiple_managers_test.py#L68